addons/mail/static/src/core/web/mail_composer_recipient_list.js

```diff
@@ -0,0 +1,37 @@
+import { registry } from "@web/core/registry";
+import { standardWidgetProps } from "@web/views/widgets/standard_widget_props";
+import { useService } from "@web/core/utils/hooks";
+
+import { Component, useState, onWillStart } from "@odoo/owl";
+import { FullRecipientsList } from "@mail/core/web/full_recipients_list";
+
+
+export class MailComposerRecipientList extends Component {
+    static template = "mail.MailComposerRecipientList";
+    static components = {
+        FullRecipientsList,
+    };
+    static props = {
+        ...standardWidgetProps,
+    };
+    setup() {
+        const { Thread } = useService("mail.store");
+        this.state = useState({});
+        onWillStart(async () => {
+            try {
+                const resIds = JSON.parse(this.props.record.data.res_ids);
```

Comment:
When there are multiple ids, you are in mutli mode, which means recipients makes no sense. It will be dynamically rendered. It should be skipped as there is no point doing any computation. same if you have a res_domain.